operator/templates/stateful-set.yaml

```diff
@@ -133,6 +135,63 @@ spec:
             - name: jvm-options
               mountPath: /etc/cassandra/jvm.options
               subPath: jvm.options
+        {{ if eq .Params.METRICS_ENABLED "true" }}
+        - name: metrics
+          imagePullPolicy: Always
+          image: viivek46/cassandra:3.11.4
+          securityContext:
+            capabilities:
+              add:
+                # TODO(mpereira): add an operator parameter for "memory lock" so
+                # that this capability and a respective Docker image with
+                # `setcap`ed binaries are conditionally used.
+                - IPC_LOCK
+                # FIXME(mpereira): might not be necessary given DCOS-59219.
+                - SYS_RESOURCE
```

Comment:
Let's remove this entire `securityContext` block for this container, it doesn't need it.

Reply:
@mpereira we currently need those capabilities to run our commands.

Reply:
Which commands?

Reply:
https://github.com/mesosphere/kudo-cassandra-operator/blob/metrics/operator/templates/stateful-set.yaml#L157
this is error we get if we dont use security context : 
`bash: /usr/bin/java: Operation not permitted`

Reply:
That's because the prometheus exporter is using the Cassandra image. It should use its own image: https://github.com/mesosphere/kudo-cassandra-operator/pull/4#discussion_r337509622

Reply:
Correct, with new images it wont be required, but currently it was required in cassandra image,  but i didnt updated this PR yet because of : https://github.com/mesosphere/kudo-cassandra-operator/pull/4#discussion_r337733759